robot_er/gazebo_ros_control/pid_gains: 
  Slider_1: {p: 50000.0, i: 0.0, d: 5000.0}
  Revolute_2: {p: 40000.0, i: 0.0, d: 5000.0}
  Revolute_3: {p: 35000.0, i: 0.0, d: 5000.0}
  Slider_4: {p: 30000.0, i: 0.0, d: 5000.0}
  Revolute_6: {p: 30000.0, i: 0.0, d: 5000.0}
  Revolute_1: {p: 50000.0, i: 0.0, d: 5000.0}

  #gripper__gripper_left: {p: 1000.0, d: 50, i: 10}
  #gripper__gripper_right: {p: 1000.0, d: 50, i: 10}
